packages/web/components/cards/stake-dashboard.tsx

```diff
@@ -106,16 +106,32 @@ export const StakeDashboard: React.FC<{
           validators={validators}
           usersValidatorsMap={usersValidatorsMap}
         />
-        <div className="flex h-full w-full flex-grow flex-row space-x-2">
+        <div className="flex h-full max-h-[150px] w-full flex-grow flex-row space-x-2">
```

Comment:
Thoughts on preferring REM values going forward? To better accommodate user font size.